docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -68,7 +68,20 @@ Each action type exposes different properties. For example an email action allow
 [role="screenshot"]
 image::images/alert-flyout-action-details.png[UI for defining an email action]
 
-Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass alert values at the time a condition is detected to an action. Available variables differ by alert type, and the list of available variables can be accessed using the "add variable" button.
+[float]
+==== Action variables
+Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass alert values at the time a condition is detected to an action. The list of available variables can be accessed using the "add variable" button. While available variables differ by alert type, all alert types pass the following variables:
```

Comment:
```suggestion
Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass alert values at the time a condition is detected to an action. You can access the list of available variables using the "add variable" button. Although available variables differ by alert type, all alert types pass the following variables:
```

Reply:
I don't see an "add variable" button in the screenshots.  Is it the + icon?

Reply:
@gchaps Yes, it is the plus icon in the screenshot.